src/validators.js

```diff
@@ -0,0 +1,514 @@
+// Functions implementing the validators of SHACL-JS
```

Comment:
Functions in this file were directly extracted from `dash.js`. Apart from applying `standard` and injecting the `$context` in all of them, I didn't do any changes.